packages/strapi-admin/admin/src/translations/index.js

```diff
@@ -44,4 +44,28 @@ const trads = {
   zh,
 };
 
+export const languageNativeNames = {
+  ar: 'العربية',
+  cs: 'čeština',
+  de: 'Deutsch',
+  en: 'English',
+  es: 'español',
```

Comment:
Can you uppercase the first letter of the native language name when possible?

Reply:
https://www.google.com.hk/preferences#languages 
I referred to this translation.  Not all regions need to be uppercased

Reply:
![Jietu20200812-105031](https://user-images.githubusercontent.com/19680644/89969778-d75c4480-dc89-11ea-8b12-ef6d5a66b979.jpg)